artifacts/snip-plugins-production-readiness.adoc

```diff
@@ -0,0 +1,13 @@
+[id='snip-plugins-production-readiness_{context}']
+[NOTE]
+====
+The following plugins have currently been vetted for RBAC/Audit logging capabilities: 
```

Comment:
```suggestion
The following plugins are currently vetted for RBAC and audit logging capabilities: 
```
Just because it sounds weird to use the word "currently" with "have been" 
Also not sure about the word "vetted" wrt translation, but nothing advised against it.

Reply:
how about, just simpler: 

```
The following plugins support  RBAC and audit logging capabilities: 
```